### ISSUE TITLE
Review and discuss adiwg_evaluationMethod codelist

### ISSUE THREAD

**Issue by hmaier-fws** [adiwg/mdCodes#58]
This issue is a placeholder to review and discuss any outstanding issues related to the adiwg_evaluationMethod.yml codelist (see original isssue #40 and PR #53). It seems that this codelist is intended to represent the `DQ_EvaluationMethod` class names, however, in a more user friendly manner, intended to populate a mdEditor pick-list.

## Items for review/discussion

  - [ ] DQ_IndirectEvaluation is not in codelist
  - [ ] DQ_EvaluationMethod is in the codelist, is that correct?

See:
  - https://wiki.esipfed.org/DQ_EvaluationMethod
  - https://standards.iso.org/iso/19157/-2/mdq/1.0/dataQualityEvaluation.xsd

## DQ_IndirectEvaluation

The codelist does not include a `DQ_IndirectEvaluation` equivalent, however, it is described in the schema. See:
  - https://wiki.esipfed.org/DQ_IndirectEvaluation
  - https://standards.iso.org/iso/19157/-2/mdq/1.0/dataQualityEvaluation.xsd
  - https://github.com/adiwg/mdJson-schemas/issues/226

## DQ_EvaluationMethod

The `DQ_EvaluationMethod` seems to be an abstract parent class that must be one of:
  - mdq:DQ_AggregationDerivation
  - mdq:DQ_FullInspection
  - mdq:DQ_SampleBasedInspection
  - mdq:DQ_IndirectEvaluation

Can DQ_EvaluationMethod stand on it's own without specifying any of the above? If not, should it be in the codelist?

**Comment by dwalt** [comment]
Yes, EvaluationMethod is needed as a class. All the sub-classes are conceptual "type" classes. Only SampleBasedInspection contains elements: samplingScheme, lotDescription and samplingRatio. As a result, Stan denormalized these elements at the EvaluationMethod level. EvaluationMethod does not belong in the codelist, only the sub-classes. See https://github.com/adiwg/mdJson-schemas/issues/226.

DataEvaluation is missing, please add. 19157:2023 has removed prefixes from class names. The current list is as follows:

- DataEvaluation
- AggregationDeriviation
- FullInspection
- SampleBasedInspection
- IndirectEvaluation
